src/Arcus.Observability.Telemetry.Core/Logging/RequestLogEntry.cs

```diff
@@ -28,10 +28,44 @@ public class RequestLogEntry
         ///     Thrown when the <paramref name="statusCode"/>'s status code is outside the 100-599 inclusively,
         ///     the <paramref name="duration"/> is a negative time range.
         /// </exception>
+        public RequestLogEntry(
+            string method,
+            string host,
+            string uri,
+            int statusCode,
+            TimeSpan duration,
+            IDictionary<string, object> context) : this (method, host, uri, $"{method} {uri}", statusCode, duration, context)
+        {
+            Guard.For<ArgumentException>(() => host?.Contains(" ") == true, "Requires a HTTP request host name without whitespace");
+            Guard.NotLessThan(statusCode, 100, nameof(statusCode), "Requires a HTTP response status code that's within the 100-599 range to track a HTTP request");
+            Guard.NotGreaterThan(statusCode, 599, nameof(statusCode), "Requires a HTTP response status code that's within the 100-599 range to track a HTTP request");
+            Guard.NotLessThan(duration, TimeSpan.Zero, nameof(duration), "Requires a positive time duration of the request operation");
```

Comment:
Since we call the other constructor we can leave these out

Reply:
Changed, see: https://github.com/arcus-azure/arcus.observability/pull/229/commits/7dbacede883ad62c749d89fcfc001c7d465891da